CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,22 @@
+# Code of Conduct
+
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
+
+The Intelligence Community Design System (ICDS) has adopted the [Contributor Covenant](https://www.contributor-covenant.org/). Please familiarise yourself with our pledge.
+
+## Our pledge
```

Comment:
Optional nitpick: Most COC have two categories where you have "Our Plege". "Pledge" and "Standards". 

Pledge explains how the project aspires to be inclusive etc, while "Standards" demonstrates how the "Pledge" is enforced in practice.